test/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildASelfContainedApp.cs

```diff
@@ -32,8 +32,6 @@ public void It_builds_a_runnable_output()
                     var ns = project.Root.Name.Namespace;
                     var propertyGroup = project.Root.Elements(ns + "PropertyGroup").First();
                     propertyGroup.Add(new XElement(ns + "RuntimeIdentifier", runtimeIdentifier));
-
-                    propertyGroup.Add(new XElement(ns + "AppendRuntimeIdentifierToOutputPath", "true"));
```

Comment:
Should we maybe set this to "false" to test out that the RID doesn't get appended to the output path?